changelog.md

```diff
@@ -8,6 +8,11 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.0.11 July 4, 2019
+* Fixed packaging errors caused by a rollup treeshaking [bug](https://github.com/rollup/rollup/issues/2970).
+* Fixed the customeditor component not able to get data from the dialog api.
```

Comment:
`not able` -> `not being able`? tenses.

Reply:
`being` is superfluous in that case and Andy has picked up on that a lot in my previous PRs :)